docs/architecture/testing/psa-storage-format-testing.md

```diff
@@ -12,7 +12,7 @@ Interoperability is not a goal: PSA crypto implementations are not intended to h
 
 ### Limitations of a direct approach
 
-The goal of storage format stability testing is: as a user of Mbed TLS, I want to store a key under version V and read it back under version W, with W ≥ V.
+The goal of storage format stability testing is: as a user of Mbed TLS, I want to store a key under version V and read it back under version W, with W >= V.
```

Comment:
If we generate PDFs using HTML as an intermediary instead of LaTeX, we should be able to have the nicer `≥` sign in the documentation